### ISSUE TITLE
Polychromatic ebuild broken 

### ISSUE THREAD

**Issue by drsn0w** [vifino/vifino-overlay#2]
After installing polychromatic using the overlay, I receive the following error when trying to run `polychromatic-controller`

```
Traceback (most recent call last):
File "/usr/bin/polychromatic-controller", line 266, in start
w.set_icon_from_file(os.path.join(path.data_source, '../install/hicolor/16x16/apps/polychromatic-controller.svg'))
GLib.Error: g-file-error-quark: Failed to open file '/usr/share/polychromatic/../install/hicolor/16x16/apps/polychromatic-controller.svg': No such file or directory (4)

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
File "/usr/bin/polychromatic-controller", line 2579, in <module>
app.start()
File "/usr/bin/polychromatic-controller", line 269, in start
w.set_icon_from_file('/usr/share/icons/hicolor/16x16/apps/polychromatic-controller.svg')
GLib.Error: g-file-error-quark: Failed to open file '/usr/share/icons/hicolor/16x16/apps/polychromatic-controller.svg': No such file or directory (4)
```

However, when installing from the Polychromatic git repo using the install script provided, everything works fine. Perhaps just making the ebuild call the included install script would fix this issue

**Comment by vifino** [comment]
I'll try to fix it, should be just a matter of adding the file path to the ebuild.
Or rather, tracing the changes in the upstream install script.

Regarding the install script, it is sadly unusable in its current state. It hardcodes paths and unless we patch it before hand, it's no good.

**Comment by vifino** [comment]
Fixed in 722871a.
